src/types/apiResponse.d.ts

```diff
@@ -1,3 +1,14 @@
+import { AnyObject } from "yup";
+
 export interface LxdApiResponse<T> {
   metadata: T;
 }
+
+export interface LxdSyncResponse {
+  type: "sync";
+  status: string;
+  status_code: number;
+  error_code: number;
+  error: string;
+  metadata: AnyObject;
```

Comment:
Surprised to see AnyObject from the yup package here. Maybe better use a generic type `LxdSyncResponse<T>` and `metadata: T`?